feedback/migrations/0020_remove_feedbacktag_feedbacks_conversation_and_more.py

```diff
@@ -7,26 +7,30 @@
 def generate_conversations_for_old_feedback(apps, schema_editor):
     Conversation = apps.get_model("feedback", "Conversation")
     Feedback = apps.get_model("feedback", "Feedback")
-    for fb in Feedback.objects.only('exercise', 'student', 'conversation'):
-        conv, _created = Conversation.objects.get_or_create(
-            exercise=fb.exercise,
-            student=fb.student,
-        )
+    fbs = list(Feedback.objects.only("exercise", "student", "conversation"))
+    new_convs = {}
+    for fb in fbs:
+        conv = new_convs.get((fb.exercise.pk, fb.student.pk))
+        if conv is None:
+            conv = Conversation(exercise=fb.exercise, student=fb.student)
+            new_convs[(fb.exercise, fb.student)] = conv
```

Comment:
uhmmm... after changing the dictionary key on line 13, you have to change it on line 16 too, right?

Reply:
done